cl/people_db/import_judges/populate_fjc_judges.py

```diff
@@ -641,3 +654,94 @@ def make_federal_judge(item, testing=False):
                 position.save()
             except Exception:
                 continue
+
+
+def make_mag_bk_judge(item, testing=False):
+    """Takes the federal judge data <item> and associates it with a Judge
+    object.  Returns a Judge object.  To be used for importing bankruptcy
+    and magistrate judges.  Example data below:
+
+    CL_ID: fjc-bk-12
+    COURT: AR,E
+    JUDGE_NAME: JONES, PHYLLIS M.
+    POSITION: Bankruptcy
+    NAME_LAST: Jones
+    NAME_FIRST: Phyllis
+    NAME_MIDDLE: M.
+    NAME_SUFFIX:
+    GENDER: Female
+    START_DATE: 2015-01-07
+    START_DATE_GRANULARITY: %Y-%m-%d
+    SOURCE: https://www.uscourts.gov/judicial-milestones/phyllis-m-jones
+    """
+
+    name = "%s: %s %s" % (
+        item["CL_ID"],
+        item["NAME_FIRST"],
+        item["NAME_LAST"],
+    )
+    print("Now processing: %s" % name)
+
+    if not pd.isnull(item["NAME_MIDDLE"]):
+        if len(item["NAME_MIDDLE"]) == 1:
+            item["NAME_MIDDLE"] += "."
+
+    if not pd.isnull(item["GENDER"]):
+        gender = get_gender(item["GENDER"])
+
+    if not item["NAME_SUFFIX"] == "":
+        suffix = get_suffix(item["NAME_SUFFIX"])
+    else:
+        suffix = ""
+
+    # Instantiate Judge object.
+    person = Person(
+        name_first=item["NAME_FIRST"],
+        name_middle=item["NAME_MIDDLE"],
+        name_last=item["NAME_LAST"],
+        name_suffix=suffix,
+        gender=gender,
+        cl_id=item["CL_ID"],
+    )
+
+    if Person.objects.filter(cl_id=item["CL_ID"]).exists():
+        raise ValidationError("CL_ID already exists.")
+
+    if Person.objects.filter(
+        name_first=item["NAME_FIRST"],
+        name_middle=item["NAME_MIDDLE"],
+        name_last=item["NAME_LAST"],
+    ).exists():
+        raise NameExistsError("")
+
+    if not testing:
+        person.save()
+
+    # Add position.
+    if re.search("Bankruptcy", item["POSITION"]):
+        position_type = "jud"
+        court = BANKRUPTCY_COURTS.get(item["COURT"])
+        print(court)
+    else:
+        position_type = "mag"
+        court = DISTRICT_COURTS.get(item["COURT"])
```

Comment:
Why use `get` for these? That prevents a KeyError, but it really just moves it along. Instead of getting a KeyError here, you get a ValidationError later when you can't create the Position object with `court=None`, right? Better to throw the error sooner, at the source?